src/panel/category/CategoryPanel.jsx

```diff
@@ -103,8 +105,23 @@ export default class CategoryPanel extends React.Component {
 
   fetchData = async () => {
     const { category, query } = this.props.poiFilters;
-
     const bbox = window.map.mb.getBounds();
+
+    // On desktop, fetch PoIs on a bbox that doesn't include the list panel's width and the header's height
+    if (!isMobileDevice()) {
+      let ne = bbox.getNorthEast();
+      let sw = bbox.getSouthWest();
+      const ne_canvas = window.map.mb.project(ne);
```

Comment:
It would be nice to put all these low level map-dependant operations somewhere else. What about a dedicated function in the `layouts` module, which would take the map instance as parameter so it doesn't create a hard dependency with `window`.

Reply:
Ok, I've put it in layouts. However, I don't think I can compute a reduced bbox without the Mapbox object (window.map.mb), so I had to keep it.